packages/vue-instantsearch/src/components/Hits.js

```diff
@@ -76,9 +82,12 @@ export default {
       );
     };
 
+    const bannerComponent =
+      bannerSlot && (({ banner }) => bannerSlot({ banner }));
```

Comment:
Thinking about it, just passing `bannerSlot` as `bannerComponent` should work, there's no need to wrap it into another function sorry for that

Reply:
Yep, good call. I just pushed up a commit directly passing it. 👍